scip.proto

```diff
@@ -135,6 +135,7 @@ message Document {
 // <identifier-character> ::= '_' | '+' | '-' | '$' | ASCII letter or digit
 // <escaped-identifier>   ::= '`' (<escaped-character>)+ '`'
 // <escaped-characters>   ::= any UTF-8 character, escape backticks with double backtick.
+// <local-id>             ::= <simple-identifier>
```

Comment:
Should we permit `<simple-identifier>` or only numeric numbers? I haven't used non-numeric IDs for local symbols, are we doing that anywhere?

Reply:
One benefit of `simple-identifier` is that the output may be more readable because you can use the name of the variable instead of only a number. So +1 to allowing simple-identifier.

Reply:
We aren't doing that anywhere ourselves, but it seems useful enough to allow for debugging in case someone wants to use letters.

Reply:
agreed 👍🏻

Reply:
Permitting more than numbers is useful for local debugging - currently when locals in document change, we renumber every single local occurrence - meaning the diff is huge.

If we did some sort of scope hashing or indexing by name, then it'd be `local <scope-id>-<relative-number>`. 

Not consequential for anything user-facing, but good for development workflow.